spring-boot-project/spring-boot-actuator/src/test/java/org/springframework/boot/actuate/metrics/web/client/MetricsRestTemplateCustomizerTests.java

```diff
@@ -157,7 +157,7 @@ void whenCustomizerAndLocalHostUriTemplateHandlerAreUsedTogetherThenRestTemplate
 
 	@Test
 	void whenAutoTimingIsDisabledUriTemplateHandlerDoesNotCaptureUris() {
-		AtomicBoolean enabled = new AtomicBoolean(false);
+		AtomicBoolean enabled = new AtomicBoolean();
```

Comment:
Just thinking out loud, but I find it more expressive in the tests to include the explicit `false`, or at least a comment indicating that the default value is false.  
It indicates intention.